lib/is-array-type.test.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+var assert = require("@sinonjs/referee").assert;
+var forEach = require("@sinonjs/commons").prototypes.array.forEach;
+var functionName = require("@sinonjs/commons").functionName;
+
+var ARRAY_TYPES = require("./array-types");
+var isArrayType = require("./is-array-type");
+
+describe("isArrayType", function() {
+    context("given an array type", function() {
+        forEach(ARRAY_TYPES, function(ArrayType) {
```

Comment:
I think that the `forEach` should wrap the `context`, and that the message should be something like `"given " + functionName(ArrayType)`, which would make the `it` message `"returns true"`